sentry_kafka_management/cli.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import argparse
+from importlib import import_module
+from typing import Mapping, Sequence
+
+FUNCTIONS: Mapping[str, str] = {
+    # function name -> module path (script) to execute
+    "get-topics": "sentry_kafka_management.scripts.topics",
+}
+
+
+def main(argv: Sequence[str] | None = None) -> int:
+    parser = argparse.ArgumentParser(
+        description="Router CLI for sentry-kafka-management. "
+        "Provide a function, remaining args are delegated to that script.",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument(
+        "function",
+        nargs="?",
+        choices=sorted(FUNCTIONS.keys()),
+        help="Function to execute (e.g., get-topics)",
+    )
+
+    # Parse only the function; leave the rest for the target script
+    args, remainder = parser.parse_known_args(list(argv) if argv is not None else None)
```

Comment:
IF we run
```
docker run --rm \
  -v "$PWD/devservices:/config" \
  sentry-kafka-management:local \
  get-topics \
  --help
```

do we see the help for that script ? That is quite important.

Reply:
> It would also be nice to see the docstrings from each function through the router. That would require importing everything beforehand.

This is what I was trying to get at. We need to be able to provide the help for each of the functions otherwise help is worthless. 

So we have a few options:
- Do not have a general `cli.py` script. Have one script per function and use a docker-entrypoint file for Docker. 
- Use python click instead of argparse that has dedicated commands https://click.palletsprojects.com/en/stable/. This library is likely going to be embedded only in internal tools so not too many places to keep up to date.
- USe subcommands with argparse https://docs.python.org/3/library/argparse.html#sub-commands